src/bridge/grok_bridge.py

```
@@ -318,6 +318,77 @@ def detect_high_thinking(prompt):
+        print(
+            f"USAGE: mode={mode} thinking={thinking} agents={agent_count} "
+            f"prompt={u.prompt_tokens} completion={u.completion_tokens} "
+            f"total={u.total_tokens} time={elapsed:.1f}s",
+            file=sys.stderr,
+        )
+        # Record to persistent usage tracker (best-effort)
+        try:
+            from usage_tracker import record_usage
+            record_usage(mode, thinking, u.prompt_tokens, u.completion_tokens, u.total_tokens, elapsed)
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)